src/js/controller/controller.js

```diff
@@ -336,13 +338,37 @@ Object.assign(Controller.prototype, {
         function _checkPlayOnViewable(model, viewable) {
             if (model.get('playOnViewable')) {
                 if (viewable) {
-                    _autoStart();
-                } else if (OS.mobile || (model.get('autoPause') && model.get('autoPause').viewability)) {
+                    _autoStart('viewable');
+                } else if (OS.mobile) {
                     _this.pause({ reason: 'autostart' });
                 }
             }
         }
 
+        function _checkPauseOnViewable(model, viewable) {
+            const instream = _this._instreamAdapter;
+
+            // If player left view and came back before ad completes,
+            // the player should begin/resume playing content.
+            if (viewable && instream && instream.noResume) {
+                instream.noResume = false;
+                model.set('playOnViewable', true);
+                return;
+            } else if (viewable || !model.get('autoPause').viewability) {
+                return;
+            }
+
+            const adState = _getAdState();
+            if (!adState) {
+                _this.pause({ reason: 'viewable' });
+            } else if (model.get('activeTab') && adState !== 'paused') {
```

Comment:
Why the check for activeTab?

Reply:
Because opening a new tab causes the viewability of the player to change. I was finding a bug in which the following behavior:

1. Click on an ad during playback. Ad pauses and a new tab opens.
2. Go back to the page with the player on it.
3. Scroll down so the player is not viewable any more.
4. Scroll back up so the player is viewable.
5. Ad will resume playback without user interaction.

Reply:
That doesn't sound like a bug to me. If you click on an ad you are not pausing the ad, the player is. Viewing the player again should resume playback (it should have resumed in step 2 so that's a different issue).

Reply:
@robwalch - I just clarified with @aliciahurst because it wasn't super-clear in the ticket and she confirmed that the behavior should remain what it is now which is that the ad remains paused when the viewer comes back to the page after clicking an ad.